src/engraving/tests/compat114_data/style-ref.mscx

```diff
@@ -4,6 +4,16 @@
     <page-offset>3</page-offset>
     <Division>480</Division>
     <Style>
+      <pageWidth>8.26772</pageWidth>
+      <pageHeight>11.6929</pageHeight>
+      <pagePrintableWidth>7.48032</pagePrintableWidth>
+      <pageEvenLeftMargin>0.393701</pageEvenLeftMargin>
+      <pageOddLeftMargin>0.393701</pageOddLeftMargin>
+      <pageEvenTopMargin>0.393701</pageEvenTopMargin>
+      <pageEvenBottomMargin>0.787403</pageEvenBottomMargin>
+      <pageOddTopMargin>0.393701</pageOddTopMargin>
+      <pageOddBottomMargin>0.787403</pageOddBottomMargin>
+      <pageTwosided>0</pageTwosided>
```

Comment:
Interesting little bug (none to fix here IMHO): while importing pageTwosided from the 1.x scores works just fine, the corresponding setting in the 4.x ref.mscx files seem to get ignored!?! Check style_ref.mscx and text_styles.mscx (they have it set), compare with the import of style.mscx and textstyles.mscx (they have it unset). No idea what might be causeing this, maybe it is limited value of plain mscx files (vs. directories) in Mu4, i.e. is is taken from an mscz file's score_style.mss, but not from a plain .mscx?

Reply:
I'm afraid I don't understand. You mean the `style-ref.mscx` and `textstyles-ref.mscx` files in `compat114_data`, right? 
Unless I'm mistaken, the tests work like this: the 1.x file is imported, and then written to some temporary `mscx` file; this temporary file is compared to the `-ref.mscx` file, using `diff`. But the `-ref.mscx` file is never read by MuseScore itself. So I don't understand what you mean with "the corresponding setting in the 4.x *.ref.mscx files seem to get ignored".

Reply:
Try reading these files in Mu4 to see what I mean.

Reply:
Ah, you mean when you open the `style-ref` file in MSS, and go to page settings, then it appears that "two-sided" is checked, while according to the file, it should be unchecked, right?

I think it is because `style-ref` is actually not a normal 4.x file, but it is written in test mode / 302 mode. When opening the file in a MSS build, it will be read in normal (non-test) mode, so the `Style` tag is ignored completely. So I think it's okay!

Reply:
OK,then...